### PR TITLE
Ensure HOC display name

### PR DESCRIPTION
**Pull request by SebCorbin**
This ensures the React component inspector is given the correct information

### CODE REVIEW

File: src/modules/Visualizer/LayersTree/index.js [hunk 1]
```diff
@@ -5,8 +5,11 @@ import UnconnectedLayersTree from './LayersTree';
 
 import { connectLayersTree } from './LayersTreeProvider/context';
 
-const LayersTree = connectLayersTree('layersTree')(UnconnectedLayersTree);
+const connectedLayersTree = connectLayersTree('layersTree')(UnconnectedLayersTree);
```
Comment (hadrienl) [inline src/modules/Visualizer/LayersTree/index.js]: Always capitalize your components.

File: src/modules/Visualizer/LayersTree/index.js [hunk 2]
```diff
@@ -5,8 +5,11 @@ import UnconnectedLayersTree from './LayersTree';
...
 
-export { connectLayersTree, LayersTreeProvider, LayersTree };
+export { connectLayersTree, LayersTreeProvider, connectedLayersTree as LayersTree };
 
-export default props => <LayersTreeProvider {...props}><LayersTree /></LayersTreeProvider>;
+const LayersTree = props => (
+  <LayersTreeProvider {...props}><connectedLayersTree /></LayersTreeProvider>
+);
+export default LayersTree;
```
Comment (hadrienl) [inline src/modules/Visualizer/LayersTree/index.js]: Rename `UnconnectedLayersTree` to ` LayersTree`, then :

```
export default props => (
  <LayersTreeProvider {...props}><connectedLayersTree /></LayersTreeProvider>
);
```

Comment (SebCorbin) [reply src/modules/Visualizer/LayersTree/index.js]: This is a bad design as it generate Anonymous components Screenshot 2019-09-02 at 19 26 33

Comment (hadrienl) [reply src/modules/Visualizer/LayersTree/index.js]: Ok!

**Pull request merged**